app/Http/Requests/Admin_OrderDetailFormRequest.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+namespace App\Http\Requests;
+
+use Illuminate\Foundation\Http\FormRequest;
+
+class Admin_OrderDetailFormRequest extends FormRequest
```

Comment:
```suggestion
class AdminOrderDetailFormRequest extends FormRequest
```
có thể tách ra thư mục nhỏ hoặc namespace nhỏ cho admin nhé, ko nên viết ntn